src/components/article/ViewArticle.jsx

```diff
@@ -77,89 +102,98 @@ class ViewArticle extends React.Component {
         </div>
 
         <div className="article-container container">
-
-          <div className={loading === '' ? 'center display-none' : 'center display-block'}>
-            {loading && <PreLoader />}
-          </div>
-          <Fragment>
-            <DisplayMessage
-              message={message}
-            />
-            <div className={typeof message !== 'string' ? 'center display-block' : 'center display-none'}>
-              <h2>{message.toString()}</h2>
-              <p>redirecting back to homepage...</p>
+          {loading ? (
+            <div className={loading === '' ? 'center display-none' : 'center display-block'}>
+              {loading && <PreLoader />}
             </div>
-            <div className={typeof message === 'string' ? 'display-block' : 'display-none'}>
-              <div className="top-details">
-                <h1 className="title">{title}</h1>
-                <p className="article-date-author">
-                  {date}
-                &nbsp;by &nbsp;
-                  {username}
-                </p>
-
-                <div className="read-time">
-                  <i className="little material-icons icon">schedule</i>
-                  <p>
-                    &nbsp;
-                    {readtime}
-                    &nbsp;
-                    read
+          ) : (
+            <Fragment>
+              <DisplayMessage
+                message={message}
+              />
+              <div className={typeof message !== 'string' ? 'center display-block' : 'center display-none'}>
+                <h2>{message.toString()}</h2>
+                <p>redirecting back to homepage...</p>
+              </div>
+              <div className={typeof message === 'string' ? 'display-block' : 'display-none'}>
+                <div className="top-details">
+                  <h1 className="title">{title}</h1>
+                  <p className="article-date-author">
+                    {date}
+                  &nbsp;by &nbsp;
```

Comment:
Please don't use `&nbsp;` to add spaces.
Modify, please.

Reply:
alright then, changed 😄